test/test_requirements.txt

```diff
@@ -0,0 +1,5 @@
+pytest
```

Comment:
I've been using `nose`, which is in the Makefile, travis ci, etc.

Reply:
How do you use it? I didn't find info about testing in the docs, so I went with the classics 😅

Reply:
Oh OK, sorry, I didn't notice the Makefile. Makes sense